test/uk/gov/hmrc/hec/controllers/AuthSupport.scala

```diff
@@ -39,14 +40,20 @@ trait AuthSupport { this: ControllerSpec =>
       .expects(predicate, retrieval, *, *)
       .returning(result)
 
-  def mockAuthWithGGRetrieval(ggCredId: String): Unit =
+  def mockGGAuthWithGGRetrieval(ggCredId: String): Unit =
     mockAuth(AuthProviders(GovernmentGateway), v2.Retrievals.credentials)(
       Future.successful(Some(Credentials(ggCredId, ggCredId)))
     )
 
-  def mockAuthWithForbidden(): Unit =
+  def mockGGAuthWithForbidden(): Unit =
     mockAuth(AuthProviders(GovernmentGateway), v2.Retrievals.credentials)(
       Future.successful(None)
     )
 
+  @silent("deprecated")
+  def mockGGOrStrideAuth(authProviderId: LegacyCredentials): Unit =
+    mockAuth(AuthProviders(GovernmentGateway, PrivilegedApplication), v2.Retrievals.authProviderId)(
```

Comment:
what's a `PrivilegedApplication`?

Reply:
it's the auth provider type which can be used to mean stride. I think it's also used to authenticate requests through MDTP's API platform (API's for external users). Privileged access is one of the modes of access we can give to external parties to our API's.